### PR TITLE
HTML FORM

### PR DESCRIPTION
**Pull request by Arinpe**
## In this project, we made an HTML FORM clone from Intuit sign in and sign up page with the following features:
- Added a sign in button on the right most part of the user interface 
- Created the form with the following types of inputs; password, e-mail and telephone
- Created a submit create account button.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+			</header>
+			<form class="form">
+				<h2 class="form-title">Create an Intuit account</h2>
+				<div class="form-para">One account for everything including Mint</div>
+				<div class="learn-more">
+					<a href="#" class="learn-more-link">learn more</a>
+				</div>
+				<div class="form-input">
+					<label for="email">email address</label>
+					<input type="text" id="email" name="email" />
```
Comment (umairarshadbutt) [inline index.html]: - The email should have an <input> of type email.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+				</div>
+				<div class="form-input">
+					<label for="number">phone (recommended)</label>
+					<input type="tel" id="number" name="number" />
```
Comment (umairarshadbutt) [inline index.html:86]: - The phone should have an <input> of type tel.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,252 @@
+/* * {
```
Comment (umairarshadbutt) [inline style.css]: Remove the commented code

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have some few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.

- [ ] Remove the comments from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)

- [ ] The email should have an <input> of type email.

- [ ] The phone should have an <input> of type tel.

- [ ] Manage the passord input tag by decreasing the padding-right 
 - **Hint:** ```padding-right: 40px;```


|[Source Page](https://accounts.intuit.com/signup.html)|[Clone Page](https://rawcdn.githack.com/Arinpe/html-forms/cffd7badb8a5732a55731aea93c673d40f0fef81/index.html)|
|:---:|:---:|
|source-page|clone|



##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳
I am very happy to see this project and I can see you worked hard on this project but I am sorry to inform you that I cannot approve this project because it is not completed yet.😟 You need to work more on it. 
_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**

- [ ] linter.yml is empty so no check is working. Please use linter files present [here](https://github.com/microverseinc/linters-config/tree/master/html-css).



**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Pull request closed without merging**